README.md

```diff
@@ -1,71 +1,65 @@
-# TVM Tutor Version Manager
-
-
-Es un comando para controlar la versión de tutor disponible en el sistema y en el virtualenv
+# What's TVM
+TVM is the acronym of Tutor Version Manager. It manages Tutor versions and enables switching between them.
 
+# Installing TVM
+## Pypi
+```bash
+pip install tutor-version-manager
```

Comment:
Is this possible already? I tried and it failed.

https://pypi.org/project/tutor-version-manager/

Reply:
@felipemontoya removed.